### DISCUSSION TITLE
Remote MongoDb Collection called 'test'

### DISCUSSION THREAD

**Discussion by peterhoang** [danny-avila/LibreChat#2316] [Q&A]
Not sure if that's normal, but when I configure a remote mongodb instance, LibreChat created a collection named 'test'. Is this by default or did I miss something in my configs?

**Comment by fuegovic** [comment]
It is the default, but you can name your database by updating your mongo URI.
The URI format is `mongodb+srv://<username>:<password>@<host>/<database>?<options>`

Be aware that the data won't transfer from `test` to the named db if you update your connection string
